Formula/f/fasmg.rb

```diff
@@ -0,0 +1,92 @@
+class Fasmg < Formula
+  desc "New assembly engine designed as a successor of the one used by flat assembler 1"
+  homepage "https://flatassembler.net"
+  url "https://flatassembler.net/fasmg.kp60.zip"
+  version "kp60"
```

Comment:
that is an interesting versioning scheme, any docs about it?

Reply:
It documented in [this post](https://board.flatassembler.net/topic.php?t=19513). It is a timestamp divided by 50 in base-36